samples/subsys/tracing/src/tracing_user.c

```diff
@@ -4,7 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <tracing_user.h>
+#include "tracing_user.h"
```

Comment:
this line probably can be left unchanged
```suggestion
#include <tracing_user.h>
```